docs/docs/en/guide/howto/datasource-setting.md

```diff
@@ -72,9 +72,10 @@ echo "host    dolphinscheduler   {user}    {ip}     md5" >> $PGDATA/pg_hba.conf
 pg_ctl reload
 ```
 
-Then, modify `./bin/env/dolphinscheduler_env.sh`, change {user} and {password} to what you set in the previous step.
+Then, set the database configurations by exporting the following environment variables, change {user} and {password} to what you set in the previous step.
 
 For MySQL:
+
```

Comment:
Do we have some duplicates here from L75-L86 with L11-L18? BTW, we have `export SPRING_DATASOURCE_URL="jdbc:postgresql://127.0.0.1:5432/dolphinscheduler"` here but we don't have this in L75-L86

Reply:
@EricGao888 thanks, this is fixed, I just adjust the doc according to the changes in this PR, didn't try to review the doc and reorganize them.